CONTRIBUTING.md

```diff
@@ -2,19 +2,44 @@
 
 This section is still work in progress
 
+## Installation
+
+We use `poetry` to manage the dependencies and the virtual environment, so you need to install it first; check the [official documentation](https://python-poetry.org/docs/#installation) for more details.
+
+Clone the repository and install the dependencies:
+
+```text
+git clone https://github.com/haddocking/arctic3d.git && cd arctic3d
+poetry install
+```
+
+OR with `pip`
+
+```text
+git clone https://github.com/haddocking/arctic3d.git && cd arctic3d
+pip install .
+```
+
 ## Testing
 
 ```text
-pip install pytest
-pytest
+run pytest --cov=./ --cov-report=xml -v
```

Comment:
the `run` key should be outside the text, right?

What if the user does use pip to install the package (without installing poetry)? I would leave the standard pytest line here for testing

Reply:
Oh yes, it's in the wrong place it, will fix it

Reply:
done, check the new version

Reply:
now the testing runs smoothly even if the user does not use poetry, nice! I guess this can be merged :)

Reply:
yes! poetry is more to help us when adding dependencies and solving the conflicts, for the users it does not matter much

Reply:
indeed!